templates/openstack-userdata.sh.j2

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+hostnamectl set-hostname github-runner
```

Comment:
Would add self-hosted somewhere in the name be better?
I fine either way just a suggestion

Reply:
I was thinking of having the shortest hostname possible - let's see if this would be fine and if users get confused we can update it later :)